floating-buttons.html

```diff
@@ -9,14 +9,16 @@
 				box-sizing: border-box;
 				display: block;
 			}
-			:host > .container {
+			.container {
```

Comment:
They end up getting scoped with `.vui-floating-buttons-0` when shimmed with the Shady-DOM, so there isn't really a concern about these styles affecting other things.  I think I am more concerned with potentially something else affecting  the children of vui-floating-buttons, and I think it would be valuable to be more specific to avoid that potential scenario.  Note this can happen regardless of whether `:host` is used.  I will make them more specific.

Note: I encountered some invalid selectors being generated using Shady-DOM, oddly only in legacy.  Specifically, selectors containing consecutive `..` and appending incorrect qualifiers to selectors.  Same code in MVC was exactly as expected.